File: solutions/cpp/count_vowels.cpp
#include <iostream>
#include <string>
using namespace std;

int countVowels(string s) {
    string vowels = "aeiouAEIOU";
    int count = 0;
    for (char c : s)
        if (vowels.find(c) != string::npos) count++;
    return count;
}

int main() {
    cout << countVowels("apple");
}

File: solutions/java/count-vowels.java
public class CountVowels {
    public static int countVowels(String s) {
        String vowels = "aeiouAEIOU";
        int count = 0;
        for (char c : s.toCharArray()) {
            if (vowels.indexOf(c) != -1) count++;
        }
        return count;
    }
    public static void main(String[] args) {
        System.out.println(countVowels("apple"));
    }
}